source/user-portal/assets/script.js

```diff
@@ -0,0 +1,56 @@
+// Script for adding recipe card
+
+// This would be the array from Local Storage
+const recipes = [
+
+
+];
+
+const recipeData = {};
+
+window.addEventListener('DOMContentLoaded', init);
+
+async function init() {
+
+    let fetched = await fetchRecipes();
+
+    if(!fetched){
+        console.error('Failed fetch');
+        return;
+    };
+
+    createRecipeCards();
+}
+
+async function fetchRecipes() {
+    return new Promise((resolve, reject) => {
+
+        for(let i = 0; i < recipes.length; i++){
+            fetch(recipes[i])
+            .then(response => response.json())
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8429335)
```suggestion
            .then((response) => response.json())
```